app/controllers/admin/commands_controller.rb

```diff
@@ -1,5 +1,8 @@
 class Admin::CommandsController < ApplicationController
-  before_action :authorize_admin!
+  include ProjectLevelAuthorization
```

Comment:
Hummm, this CommandsController should get split between admin (gor global commands) and project specific one.
In the meantime, would it be better to have: